proto/delivery/execution.proto

```diff
@@ -50,5 +50,5 @@ message DeliveryExecution {
 
   // Version of the execution server, for example a commit hash
   // or public repo release version.
-  string ExecutionServerVersion = 7;
+  string server_version = 7;
```

Comment:
i think @scottmcmaster made a similar change...

Reply:
This is the public generated version, he probably made the change to the internal version.

Reply:
Yea, he did.

Reply:
ah... i saw schema so i didn't see this was the public one.  lgtm!